template.json

```diff
@@ -1,7 +1,7 @@
 {
   "Actor": {
     "types": [
-      "Zero-Level",
+      "Player",
```

Comment:
Currently this will break existing zeroes. :(

The options are to live with PCs being called Zero-Level forever or figure out how to write a migration that changes an actor's entity type. I'll look into the latter option soon!